wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -110,22 +110,24 @@ class KotlinGeneratorTest {
           |package routeguide
           |
           |import com.squareup.wire.GrpcCall
+          |import com.squareup.wire.GrpcClient
+          |import com.squareup.wire.GrpcMethod
           |import com.squareup.wire.Service
-          |import com.squareup.wire.WireRpc
           |
           |/**
           | * RouteGuide service interface.
           | */
-          |interface RouteGuideClient : Service {
+          |class RouteGuideClient(
+          |  private val client: GrpcClient
+          |) : Service {
           |  /**
           |   * Returns the [Feature] for a [Point].
           |   */
-          |  @WireRpc(
-          |    path = "/routeguide.RouteGuide/GetFeature",
-          |    requestAdapter = "routeguide.Point#ADAPTER",
-          |    responseAdapter = "routeguide.Feature#ADAPTER"
-          |  )
-          |  fun GetFeature(): GrpcCall<Point, Feature>
+          |  fun GetFeature(): GrpcCall<Point, Feature> = client.newCall(GrpcMethod(
+          |      path = "/routeguide.RouteGuide/GetFeature",
+          |      requestAdapter = Point.ADAPTER,
+          |      responseAdapter = Feature.ADAPTER
+          |  ))
```

Comment:
@Egorand I'd rather have something like
```kotlin
fun GetFeature(): GrpcCall<Point, Feature> = client.newCall(
   GrpcMethod(
     path = "/routeguide.RouteGuide/GetFeature",
     requestAdapter = Point.ADAPTER,
     responseAdapter = Feature.ADAPTER
  )
)
```
Is KotlinPoet doing it on purpose? or is it us?